plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -58,9 +58,13 @@ module.exports = new Transformer({
 				config.filePath.replace('.scaffoldrc', ''),
 				config.contents.templatePath
 			);
+			const tocPath = config.contents.tocPath || '';
 			const templateLocation = path.resolve(basePath, `${attributes.template}.html`);
 
 			const template = await options.inputFS.readFile(templateLocation, 'utf-8');
+			const tocEntries = tocPath
+				? await options.inputFS.readFile(tocPath, 'utf-8').then(r => JSON.parse(r))
+				: null;
 
```

Comment:
```
const [template, tocEntries] = await Promise.all([
   options.inputFS.readFile(templateLocation, 'utf-8'),
   tocPath ? options.inputFS.readFile(tocPath, 'utf-8').then(r => JSON.parse(r)) : Promise.resolve(null)
])
```

Reply:
would be good to cache the template/toc because this logic is running for each markdown file.

Reply:
scratch the cache idea due to separate processes

Reply:
diff in convo tab isn't updating for me, but the update is evident in files. there is a build error related so cross platform stuff, so I'm gonna jump on the mac and test further